DQM/Physics/python/DQMPhysics_cff.py

```diff
@@ -33,6 +34,7 @@
                            *HiggsDQM
                            *ExoticaDQM
                            *B2GDQM
+                           *CentralitypADQM
```

Comment:
if this requires products from pA era, it should be enabled only with pA era.
... I can see that the PR will be held up until #16394 is merged